packages/block-editor/src/components/block-styles/index.js

```diff
@@ -7,6 +7,8 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import isShallowEqual from '@wordpress/is-shallow-equal';
+import { memo } from '@wordpress/element';
```

Comment:
noting that we also have a "pure" hoc which is the same thing (before the existence of memo), not sure if we should deprecate it or not.

Reply:
I also find both these hocs to be less and less useful with hooks

Reply:
I have no strong preference here either - I'm fine with just `memo`, but if you think it would be cleaner with `pure` I'll update it.